src/openassets.es6

```diff
@@ -29,14 +29,16 @@ module.exports = function(robot) {
   // Address Book
   //
 
+  let robotKeyword = process.env.OA_BOT_KEYWORD || 'kredits';
```

Comment:
Shouldn't we also set the kredits asset ID by default then? If we don't do that, we might as well throw an error if the keyword isn't set I think.